### ISSUE TITLE
Error when trying to execute object line crossing pipeline

### ISSUE THREAD

**Issue by antoniomtz** [intel/video-analytics-serving#84]
Hello, 
I'm getting the following error when trying to execute the object line crossing pipeline:

```
{"levelname": "INFO", "asctime": "2021-08-18 18:45:44,824", "message": "Creating Instance of Pipeline object_tracking/object_line_crossing", "module": "pipeline_manager"}
{"levelname": "INFO", "asctime": "2021-08-18 18:45:44,828", "message": "Gstreamer RTSP Server Started on port: 8554", "module": "gstreamer_rtsp_server"}
{"levelname": "ERROR", "asctime": "2021-08-18 18:45:44,843", "message": "Error on Pipeline 1: gi.FunctionInfo(set_state) returned a result with an error set", "module": "gstreamer_pipeline"}
{"levelname": "ERROR", "asctime": "2021-08-18 18:45:44,843", "message": "Error on Pipeline 1: gst-library-error-quark: Error creating Python callback (3): /root/gst-video-analytics/gst/elements/gvapython/gstgvapython.c(210): gst_gva_python_start (): /GstPipeline:pipeline9/GstGvaPython:object-line-crossing:\nModule: /home/video-analytics-serving/extensions/spatial_analytics/object_line_crossing.py\n Class: ObjectLineCrossing\n Function: process_frame\n Arg: []\nKeyword Arg: {}\n", "module": "gstreamer_pipeline"}
```

I get a similar error when running the object zone count as well. Any ideas?

**Comment by antoniomtz** [comment]
I was able to make it run by adding `--parameter-file vaclient/parameter_files/object-line-crossing.json` found on another ticket.

**Comment by nnshah1** [comment]
Thanks Antonio! We also found late that the pipeline didn't run with default parameters and required parameters. We're working to improve that for the next release so that the pipeline will run by default (without any events) but also clarify how to set them.

**Comment by antoniomtz** [comment]
@nnshah1 Do you have an estimate date for the next release?

**Comment by nnshah1** [comment]
We are currently working on a minor update (0.6.1) to coincide with the next OpenVINO release (roughly three weeks after).

**Comment by whbruce** [comment]
Fixed in [v0.6.1](https://github.com/intel/video-analytics-serving/releases/tag/v0.6.1-beta).
